frontend/src/libs/api.ts

```diff
@@ -41,6 +13,7 @@ export default async function api(endpoint: string, options?: RequestInit) {
     ...options
   });
 
+  // why do we need this?
   if (response.status === 204) {
     return null;
   }
```

Comment:
This can be safely removed, I took this from another project and didn't remove it, my bad!